packages/server/modules/core/graph/resolvers/streams.ts

```diff
@@ -312,6 +323,10 @@ export = {
   },
 
   Stream: {
+    isPublic(parent) {
+      return parent.visibility === ProjectRecordVisibility.Public
```

Comment:
@JR-Morgan this PR may require adjustments to SDK tests. Project.visibility UNLISTED will now return PUBLIC instead. Other than that the API should be more or less the same.

Reply:
this is about access not about where you're able or not able to find them - whichever way you find the project, through link or through your dashboard, its accessible by the public (literally anyone).

public/private terminology is way more intuitive when it comes to access levels than 'unlisted'

Reply:
where the project is listed or not should be a separate setting altogether, like how we may want to introduce link shareability for any visibility through a URL token or something